lib/child-writer/writer-proxy.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 resin.io
+ * Copyright 2017 resin.io
```

Comment:
By contrast, the files in this PR were created in 2016, so I wonder if their copyright-year should stay as 2016? I'm afraid I'm not a copyright lawyer, so I dunno what the "correct" solution is ;-)

EDIT: Did a quick search, and found http://stackoverflow.com/a/2391555 and https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html which at least seem to be consistent with each other.